core/rlwe/params.go

```diff
@@ -610,12 +613,15 @@ func (p Parameters) Equal(other *Parameters) (res bool) {
 // MarshalBinary returns a []byte representation of the parameter set.
 // This representation corresponds to the MarshalJSON representation.
 func (p Parameters) MarshalBinary() ([]byte, error) {
```

Comment:
Changes and additions to the parameters serialization are still not tested (and this previously caused a merge into main of code not working as expected)

Reply:
Fixed.